### PR TITLE
editorial pass

### PR DESCRIPTION
**Pull request by bumblefudge**
Signed-off-by: bumblefudge <jcaballero@centre.io>

### CODE REVIEW

**Comment by bumblefudge** [comment]
@zachferland i can't tag you for review but you should probably check nothign I rephrased got mangled, particularly [this](https://github.com/ceramicnetwork/namespaces/pull/1/files#diff-7a8db22fd601906e16dc95503933353f67c6f453498e591e08ad490f920d36c7R72) and [this](https://github.com/ceramicnetwork/namespaces/pull/1/files#diff-7a8db22fd601906e16dc95503933353f67c6f453498e591e08ad490f920d36c7R100-R104)

File: eip155/caip168.md [hunk 1]
```diff
@@ -55,17 +81,39 @@ Example 2
...
+4) The partial CID is the **multihash portion** of the original CIDv1 (refer the
+   section [How does it
+   work?](https://github.com/multiformats/cid#how-does-it-work) in the
+   [Multiformats CID][] specification for context). It does not include the
+   multibase, the CID version or the IPLD codec segments. It is assumed that the
+   IPLD codec is dag-cbor. Verify that the first 32bytes of the `root` CID of
+   the timestamp anchor proof is equivalent to the 32-byte partial CID extracted
+   from the transaction data. Equivalence should be checked using a CIDs
+   library, which can compare oprtions of multihashes encoded in bytes. If the
+   two partial multihashes are not equivalent, verification MUST fail.
```
Comment (zachferland) [inline eip155/caip168.md]: I may have confused wording a bit using `multihash` and `partial cid` interchangeably, but it is technically just the multihash being used, so updated below to only reference that 

```suggestion
   IPLD codec is dag-cbor. Verify that the multihash portion of the `root` CID of
   the timestamp proof is equivalent to the multihash extracted
   from the transaction data. Equivalence should be checked using a CIDs
   library, which can compare multihash portions encoded in bytes. If the
   two multihashes are not equivalent, verification MUST fail.
```

File: eip155/caip168.md [hunk 2]
```diff
@@ -34,19 +43,36 @@ Example 1
...
 
-This section describes transaction verification when a timestamp anchor proof references a `chainId` in the `eip155` namespace and `txType` is `f(bytes32)`. Type `f(bytes32)` refers to a transaction which makes a function call on a contract with the function signature including a single 32 bytes argument. The 32 byte argument is the merkle root CID. A CID is more than bytes32, and a partial CID is used to allow the argument to be efficiently packed in the transaction. Function name or contract does not matter, and is up to an implementation. 
+This section describes transaction verification when a timestamp anchor proof
+references a `chainId` in the `eip155` namespace and `txType` is `f(bytes32)`.
+Type `f(bytes32)` refers to a transaction which makes a function call on a
+contract with the function signature including a single 32 bytes argument. The
+32 byte argument is the merkle-root CID. A CID is more than 32 bytes, and a
+partial CID is used to allow the argument to be efficiently packed in the
+transaction. Function name or contract does not matter, and is up to the
+implementation.  Use of a root CID encoded in [DAG-CBOR][] is recommended.
```
Comment (zachferland) [inline eip155/caip168.md:72]: 👍

**Review by zachferland — COMMENTED** [review]
thanks @bumblefudge, minor wording change to your edits, let me know if that makes sense

**Review by zachferland — APPROVED** [review]
thanks! lgtm

**Pull request merged**